src/helper/task.ts

```diff
@@ -42,6 +42,8 @@ import {
   EstimatedProofFee,
   ForceUnprovableToReprocessParams,
   ForceDryrunFailsToReprocessParams,
+  ProverNodeTimerangeStatsParams,
+  ProverTimeRangeStats,
```

Comment:
Please make the naming consistent, not one is ProverNode, another is Prover.
Please use "ProverNodeTimeRangeStats"